temporalcli/commandsgen/commands.yml

```
@@ -2886,6 +2886,9 @@ commands:
       - name: limit
         type: int
         description: Maximum number of Workflow Executions to display.
+      - name: pagesize
+        type: int
+        description: Maximum number of Workflow Executions to return from server per request. Used for pagination.
```

Optional; rewords in terms of behavior, not implementation.

```suggestion
        description: Maximum number of Workflow Executions to fetch at a time from the server.
```